charts/openyurt/templates/yurt-manager-auto-generated.yaml

```diff
@@ -804,3 +804,22 @@ webhooks:
     resources:
     - nodepools
   sideEffects: None
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: webhook-service
+      namespace: kube-system
+      path: /validate-core-openyurt-io-pod
```

Comment:
path should be /validate-core-openyurt-io-v1-pod instead of /validate-core-openyurt-io-pod